### PR TITLE
status-updates: Add summary of meeting on Friday, May 19, 2023

### PR DESCRIPTION
**Pull request by teodutu**

### CODE REVIEW

File: status-updates/2023/2023-05-19.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+## Discussions
+
+### Closing Event
+
+RD: Let's organise it on September 14.
+We'll schedule a full-day event.
+We'll first book a larger room, and then attendees will split into breakout rooms on different subjects: quizzes, assignments, etc.
+
+RD: We are looking at around 200 attendees.
```
Comment (StefanJum) [inline status-updates/2023/2023-05-19.md]:
```suggestion
RD: We are looking at around 100 attendees.
```
Wasn't it 100?

**Review by StefanJum — COMMENTED** [review]
All good besides the one comment.

File: status-updates/2023/2023-05-19.md (hunk 1 above)
Comment (teodutu) [reply status-updates/2023/2023-05-19.md:15]: @razvand ?

Comment (teodutu) [reply status-updates/2023/2023-05-19.md]: Oh, yeah, 200 invitees and 100 attendees. I'll fix that.

**Review by StefanJum — APPROVED** [review]
All good now.

**Review by lizababu — APPROVED** [review]

**Pull request merged**